w3c/aria-at
# Exclude 0-level priority assertions in the collection form

[Preview Tests](https://deploy-preview-1012--aria-at.netlify.app)

See #1008.

To test this change:
* Run `npm run make-v2 <pattern>`
* Update any of the created `*-commands.csv`'s row value for the _"assertionExceptions"_ column with `0:<assertionId> ...`
  * Note: `assertionId` must match a defined assertionId of a row in the generated `assertions.csv`
  * Note: `tests.csv` must include the assertion id in a row's _"assertions"_ column
* Run `npm run build`
* Observe that the assertion is excluded on `<baseurl>/review/<pattern>.html` under the expected command (this was included before this PR)
* Select an AT link under tests and observe that the assertion is excluded on `<baseurl>/tests/<pattern>/<testfile_name>.html` under the expected command

**Note that #1014 has also been submitted to make previewing those changes easier (check description for changes and [preview link](https://deploy-preview-1014--aria-at.netlify.app/))**
